tests/test_tutorial/test_parameter_types/test_enum/test_tutorial001.py

```diff
@@ -17,6 +17,7 @@ def test_help():
     assert result.exit_code == 0
     assert "--network" in result.output
     assert "[simple|conv|lstm]" in result.output
+    assert "default: simple" in result.output
```

Comment:
Note that the docs in fact already document this behaviour (first bash script on [enum page](https://typer.tiangolo.com/tutorial/parameter-types/enum/)):
>   --network [simple|conv|lstm]  [default: simple]

But on `master` you currently get
> --network        [simple|conv|lstm]  [default: NeuralNetwork.simple]  

So the docs don't require further updating - this PR really just ensures the correct behaviour as it was already documented.